src/main.rs

```diff
@@ -250,20 +252,60 @@ fn run_machete() -> anyhow::Result<bool> {
     Ok(has_unused_dependencies)
 }
 
-fn remove_dependencies(manifest: &str, dependencies_list: &[String]) -> anyhow::Result<String> {
+// handle a superset of all dependency name dashed/underscored variants: re'\w[-_]'
+fn dep_name_superset(dep_names: &[String]) -> HashSet<String> {
+    let mut unused: HashSet<String> = dep_names.iter().cloned().collect();
+    for dep in unused.clone() {
+        unused.insert(dep.replace('-', "_"));
+        unused.insert(dep.replace('_', "-"));
+    }
+    unused
+}
+
+fn remove_dependencies(manifest: &str, dependency_list: &[String]) -> anyhow::Result<String> {
     let mut manifest = toml_edit::Document::from_str(manifest)?;
```

Comment:
`dependency_list` seems to consistently normalize for underscores but the `toml_edit::Document` preserves multi word delimiters